src/arrayMethodJoin.js

```diff
@@ -4,9 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let resultJoin = '';
+
+    for (let i = 0; i <= this.length - 1; i++) {
+      if (this[i] !== null && this[i] !== undefined) {
+        resultJoin += `${this[i]}`;
```

Comment:
```suggestion
        resultJoin += this[i];
```
Unnecessary template literal. You need only variable to concatenate with string.